nxcompshad/src/Core.h

```diff
@@ -88,9 +88,9 @@ class CorePoller
 
   unsigned int bpl_;
 
-  unsigned int width_;
+  unsigned short int width_;
```

Comment:
This also holds true for stuff that uses it in, for instance, `Core.cpp` - like the cast to `int` on line 258, 407 and others. Note that this is dangerous even with the current code, since casting `unsigned int` to `int` unconditionally could easily overflow. Casting `unsigned short` to `int` is even worse. If this stuff is changed, `unsigned short` to `short` is better but still prone to overflowing.